.github/workflows/ci_docker.yml

```diff
@@ -37,6 +40,33 @@ jobs:
         env:
           ACCEPT_EULA: Y
           SA_PASSWORD: MyS3cr3tPassw0rd
+      rmq:
+        image: rabbitmq:3
+        ports:
+          - 5672:5672
+        env:
+          RABBITMQ_DEFAULT_USER: guest
+          RABBITMQ_DEFAULT_PASS: guest
+      ewh:
+        image: docker.pkg.github.com/sanger/event_warehouse/event_warehouse:gpl769_test_cp_view
```

Comment:
We need to remember to update the image tag.

Reply:
Ok, I'll update to the current status of merged branches (develop for uwh, master for ewh)